sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 ### Breaking Changes
 
+> These changes do not impact the API of stable versions such as 1.14.0.
+> Only code written against a beta version such as 1.15.0b1 may be affected.
+- Windows Web Account Manager (WAM) Brokered Authentication is moved into another package.
```

Comment:
Should mention the name of the package
```suggestion
- Windows Web Account Manager (WAM) Brokered Authentication is moved into another package (`azure-identity-broker`)
```

Reply:
We have not finalized the package name yet. :)

Reply:
Ah good point